zeppelin-zengine/pom.xml

```diff
@@ -650,6 +645,21 @@
           <version>${hadoop.version}</version>
           <scope>test</scope>
         </dependency>
+            <dependency>
+          <groupId>org.codehaus.jettison</groupId>
+          <artifactId>jettison</artifactId>
+          <version>${jettison.version}</version>
+        </dependency>
+            <dependency>
+          <groupId>org.apache.hadoop</groupId>
+          <artifactId>hadoop-aws</artifactId>
```

Comment:
It would probably be better to use `hadoop-cloud-storage` here since that would not only get s3, azure, and gcp as well. No need to individually pull these in.

Also I would hope that don't need kerberos client explicitly here.

Reply:
Currently zeppelin support hadoop-3.0.0 version and if we want to support hadoop-cloud-storage then we have to bump-up the hadoop version.

Reply:
I am fine to upgrade to hadoop 3.1.x

Reply:
Shall we add a new profile? or do we have to update hadoop-version from 3.0.0 to 3.1.1

Reply:
@zjffdu / @risdenk : I think we have to create separate jira to bump-up hadoop version.

Reply:
Make sense, let's do it in separate PR

Reply:
If everything is fine for you then can you please give +1.